### PR TITLE
update yaml in the multiple schedulers.md

### PR DESCRIPTION
**Pull request by jianglingxia**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7109)

### CODE REVIEW

File: docs/tasks/administer-cluster/my-scheduler.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```
Comment (tengqm) [inline docs/tasks/administer-cluster/my-scheduler.yaml]: we should instead avoid adding this comment considering that even earlier versions will need apps/v1beta1 or extensions group. In the master branch, we can safely assume users are reading docs about v1.9.0+. Docs for older supported versions can be found elsewhere.

**Comment by tengqm** [comment]
also, the gate error is real, please check.

File: docs/tasks/administer-cluster/my-scheduler.yaml (hunk 1 above)
Comment (jianglingxia) [reply docs/tasks/administer-cluster/my-scheduler.yaml]: thanks for your review,you mean that delete the comment and only modify to apps/v1?

Comment (tengqm) [reply docs/tasks/administer-cluster/my-scheduler.yaml]: yes

**Comment by jianglingxia** [comment]
I cannot understand why the ci always failed!

**Comment by tengqm** [comment]
The error message says "`selector` does not match template `labels`". This is because in apps/v1, the Deployment has to explicitly specify selector and labels and they have to match each other.

**Comment by heckj** [comment]
Could you please review that recommendation and update these PRs to [include min-kubernetes-server-version in the front-matter of the various documents](https://kubernetes.io/docs/home/contribute/style-guide/#versioning-kubernetes-examples)?

**Comment by jianglingxia** [comment]
the docs has correct, and close it!

**Pull request closed without merging**